lib/handlers/infoHandlerFactory.js

```diff
@@ -0,0 +1,30 @@
+const {
+  abci: {
+    ResponseInfo,
+  },
+} = require('abci/types');
+
+/**
+ * info ABCI handler
+ * @param {AppState} state
+ * @returns {infoHandler}
+ */
+module.exports = function infoHandlerFactory(state) {
+  /**
+   * @typedef infoHandler
+   * @returns {Promise<ResponseInfo>}
+   */
+  async function infoHandler() {
+    const lastBlockHeight = state.getHeight();
+    const lastBlockAppHash = state.getAppHash();
+
+    return new ResponseInfo({
+      data: 'Dash ABCI JS State Machine',
+      version: '0.1.0',
+      lastBlockHeight,
```

Comment:
We need to add AppVersion too. It must be DPP version (from `@dashevo/dpp` package).